src/renderer/entities/price/model/currency-model.ts

```diff
@@ -8,6 +8,14 @@ const $currencyConfig = createStore<CurrencyItem[]>([]);
 const $activeCurrency = createStore<CurrencyItem | null>(null);
 const $activeCurrencyCode = createStore<CurrencyItem['code'] | null>(null);
 
+const $cryptoCurrencies = $currencyConfig.map((config) => config.filter((c) => c.category === 'crypto'));
```

Comment:
These derived stores should be in `form-model`, entity must contain most generic data and logic for it's transformation like: 
- config / changeConfig
- activeCurrency / changeActiveCurrency
and so on